gildas: init at 201806_a

smaret opened the pull request:
###### Motivation for this change

This PR provides a package for [Gildas](https://www.iram.fr/IRAMFR/GILDAS/), a collection of software for radio-astronomy. It's a re-submission of PR #27138 with an updated upstream version (201806_a), Python support ~~and OpenBLAS support~~.

The package has been tested on Linux and Darwin.

###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [x] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

smaret committed:
gildas: init at 201806_a

smaret commented:
CC @bzizou

ryantm commented on pkgs/applications/science/astronomy/gildas/default.nix:
```diff
@@ -0,0 +1,68 @@
+    make install
+  '';
+
+  installPhase=''
```
Can this be moved to the `postInstallPhase`? Then it looks like the standard `installPhase` can call `make install`, which is currently in the `buildPhase`.

smaret replied:
Done in 8f8a251

smaret committed:
gildas: use the standard installPhase

smaret committed:
gildas: use internal LAPACK instead of OpenBLAS

Gildas' uv_fit task segfaults when it uses OpenBLAS. Use internal
LAPACK version instead.
